muuvarastopaikka.php

```diff
@@ -559,6 +569,24 @@
           $siirretaan[$iii] = $lisavartprow["tunnus"];
         }
       }
+      // P�ivitet��n uusi paikka varatuille tilausriveille
+      if ($siirretaan_varattua) {
+        $query = "UPDATE tilausrivi
+                  SET hyllyalue = '$minnerow[hyllyalue]',
+                    hyllynro    = '$minnerow[hyllynro]',
+                    hyllyvali   = '$minnerow[hyllyvali]',
+                    hyllytaso   = '$minnerow[hyllytaso]'
+                  WHERE tuoteno   = '$tuotteet[$iii]'
+                    AND yhtio     = '$kukarow[yhtio]'
+                    AND tyyppi    = 'L'
+                    AND varattu > 0
```

Comment:
@mikamyllynen @tonyselkamo AND varattu > 0. Miten käy varattu < 0 riveille? Pitääkö nekin päivittää vai halutaanko nimenomaan päivittä vain > 0 rivit?